config/locales/js-en.yml

```diff
@@ -231,7 +231,7 @@ en:
       label_search_watchers: Search watchers
       label_add: Add watchers
       label_discard: Discard selection
-      label_click_to_add: Click to add watchers
+      tyepahead_placeholder: Search for possible watchers
```

Comment:
You 'tyep'-edahead a little to fast here. But at least it is consistent with https://github.com/opf/openproject/pull/5054/files#diff-84034d217bbf167c6ffdbcd55ffa6bdaR57